eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java

```diff
@@ -355,6 +362,7 @@ private void checkIfCandidateForRSSplit(SubmittedVariantEntity submittedVariantE
                         "Hash mismatch with " + variantAccession, inactiveEntities);
 
                 rsSplitCandidateSVOE.put(variantAccession, submittedVariantOperationEntity);
+                updateRsSplitCandidateSVOE.put(variantAccession, submittedVariantOperationEntity);
```

Comment:
This map should be populated [here](https://github.com/EBIvariation/eva-accession/blob/614bdfe8a91dccdffec1195738f9225805d3a4cf/eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/batch/io/ClusteringWriter.java#L352) too...